frontend/src/libs/components/meditation-tracks-list/mock/mocked-tracks.ts

```diff
@@ -0,0 +1,34 @@
+const mockedImageLink = './images/track-background-336x480.jpg';
```

Comment:
use it as placeholder, don't pass as mocked image
```suggestion
```

Reply:
i mean to pass it in place where you render items if image is not present in data

Reply:
fixed(hope I'v undestood it correctly)